src/rules/social/infoboard.js

```diff
@@ -0,0 +1,34 @@
+import { interval } from '../helpers';
+import { logger } from '../../logger';
+import { InfoEntry } from '../../models/infoentry';
+
+const POLL_FREQUENCY_MS = 10000;
+
+const closeInfoentryTimers = new Map();
+
+function closeInfoentry(infoentry) {
+	logger.info('Closing Infoentry', infoentry.get('id'));
+	return infoentry.save({ enabled: false }, { method: 'update', patch: true });
+}
+
+async function updateInfoentryScheduledToClose() {
+	const activeInfoentry = await new InfoEntry().where('enabled', true).where('active_until', '!=', '').fetchAll();
```

Comment:
Probably should check for non-null values instead of non-empty strings:
```suggestion
	const activeInfoentry = await new InfoEntry().where('enabled', true).whereNotNull('active_until').fetchAll();
```

Reply:
Tested, doesn't work. Crashes the backend.

```
        const activeInfoentry = await new InfoEntry().where('enabled', true).whe
reNotNull('active_until').fetchAll();
                                                                      ^
TypeError: (intermediate value).where(...).whereNotNull is not a function
    at updateInfoentryScheduledToClose (/workspace/src/rules/social/infoboard.js
:15:71)
```